### PR TITLE
feature/add-legislation-card

### LINKED ISSUE #4: Legislation card
Related to  [#79](https://github.com/CouncilDataProject/seattle/issues/79).
Write a legislation card component and update the styling to match the UX team's designs.

### PR DESCRIPTION
**Pull request by tohuynh**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [x] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
  Resolves #4 
- [x] Provide context of changes.
  Add the legislation card component and its stories to storybook.
  In the legislation card, there are different colored(using cdp-design colors) icons to designate the status of a legislation.
  [Preview](https://tohuynh.github.io/cdp-frontend/?path=/story/library-cards-legislation--adopted)
- [ ] Provide relevant tests for your feature or bug fix.
- [ ] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

**Review by JacksonMaxfield — APPROVED** [review]
This all looks great to me. Hopefully we can fix the broken build though. May want to look at `cdp-design`'s webpack loader config?

**Comment by tohuynh** [comment]
> This all looks great to me. Hopefully we can fix the broken build though. May want to look at `cdp-design`'s webpack loader config?

Awesome, that fixed it. Should I separate out the rule for font files(woff) into its own rule?

**Review by JacksonMaxfield — APPROVED** [review]
Merge whenever!

File: src/components/Cards/LegislationCard/LegislationCard.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    statusIcon = <LegislationAdoptedIcon />;
+  } else if (status === "Rejected") {
+    statusIcon = <LegislationRejectedIcon />;
+  } else {
+    statusIcon = <LegislationInProgressIcon />;
+  }
+
+  return (
+    <section className="mzp-c-card mzp-has-aspect-16-9">
+      <div className="mzp-c-card-block-link">
```
Comment (hawkticehurst) [inline src/components/Cards/LegislationCard/LegislationCard.tsx:48]: What was the reason that this card-block-link element was turned into a div and is no longer a link element?

Seems to deviate from Protocol documentation examples:
https://protocol.mozilla.org/patterns/molecules/card.html

**Review by hawkticehurst — APPROVED** [review]
Beyond my question about the `card-block-link` element, this all looks great!

File: src/components/Cards/LegislationCard/LegislationCard.tsx (hunk 1 above)
Comment (hawkticehurst) [reply src/components/Cards/LegislationCard/LegislationCard.tsx:48]: I just read through the comments you gave me in my PR and realized you already answered this exact question!

Keeping card routing unopinionated sounds perfectly reasonable to me.

Going to resolve this conversation and feel free to merge whenever!

**Pull request merged**